src/dart_diagnostic_provider.ts

```diff
@@ -17,7 +17,7 @@ export class DartDiagnosticProvider {
 
 	private handleErrors(notification: as.AnalysisErrorsNotification) {
 		let errors = notification.errors;
-		if (!getConfig('showTodos'))
+		if (!getConfig<boolean>('showTodos'))
```

Comment:
> I'll probably pull them all together soon into a strongly-typed class.

Sounds great - I have the same string literal (`showTodos`) in 2-3 places now; having a class for the pref name would be more robust.